@xen-orchestra/backups/_runners/_vmRunners/FullXapi.mjs

```diff
@@ -35,13 +35,23 @@ export const FullXapi = class FullXapiVmBackupRunner extends AbstractXapi {
         useSnapshot: false,
       })
     )
+
+    const vdis = await exportedVm.$getDisks()
+    let maxStreamLength = 1024 * 1024 // Ovf file and tar headers are a few KB, let's stay safe
+    vdis.forEach(vdiRef => {
+      const vdi = this._xapi.getObject(vdiRef)
```

Comment:
Prefer using `getRecord` over `getObject` in `@xen-orchestra/backups`.

Current code is not perfect as this regard but I'd like this lib (as well as `xen-api` and `@xen-orchestra/xapi`) without having to fetch all records in memory first.